CalPatRec/src/CalHelixFinder_module.cc

```diff
@@ -61,6 +61,7 @@ namespace mu2e {
     _tpart              ((TrkParticle::type)(pset.get<int>("fitparticle"))),
     _fdir               ((TrkFitDirection::FitDirection)(pset.get<int>("fitdirection"))),
     _doSingleOutput     (pset.get<bool>  ("doSingleOutput")),
+    _maxEDepAvg            (pset.get<float>  ("maxEDepAvg")),
```

Comment:
Use of ParameterSet for module implementation is deprecated. As per Mu2e software policy established several years ago this module must be upgraded to use validated fhicl before this PR can be processed.